export/planloader/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: Apache-2.0
+
+set(CMAKE_POSITION_INDEPENDENT_CODE TRUE)
+
+if(NOT BUILD_SUBDIR_NAME EQUAL "release")
+  message(
+    SEND_ERROR,
+    "The planloader library does not work in Debug mode due to its dependencies."
+  )
+endif()
+
+add_library(planloader SHARED planloader.cpp)
+set_target_properties(planloader PROPERTIES SUBVERSION 1)
```

Comment:
?

Reply:
This supposedly sets the minor version of the shared library but that value is the default so I'm eliminating it here.